docs/self-managed/components/orchestration-cluster/identity/connect-external-identity-provider.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Connect Orchestration Cluster to Identity Provider
+sidebar_label: Connect Orchestration Cluster to Identity Provider
+description: Learn how to connect Camunda 8 Orchestration Cluster Identity to an external Identity Provider (IdP) via OpenID Connect (OIDC) for authentication and user management.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Connect Orchestration Cluster to Identity Provider
+
+This guide explains how to configure the Orchestration Cluster to use an external Identity Provider (IdP) via OpenID Connect (OIDC) for authentication and user management.
+
+**User Interface access:**
+
+Users authenticate via the **OIDC Authorization Code Flow**. This means users are redirected to your Identity Provider to log in, and Camunda receives a token to establish the session.
```

Comment:
💭 Should here instead we just link externally to the spec about Authorizations code flow? https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth - I think that for users reading this guide just knowing
> Users authenticate via the **OIDC Authorization Code Flow**.

is sufficient IMO

Reply:
good idea, resolved